docs/blog/_posts/2018-04-27-eighth-dotty-milestone-release.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.7.0 and 0.8.0-RC1
+author: Allan Renucci
+authorImg: /images/allan.jpg
+date: 2018-04-27
+---
+
+Today, we are excited to release Dotty versions 0.7.0 and 0.8.0-RC1. These releases
+serve as a technology preview that demonstrates new language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
+pioneered the fusion of object-oriented and functional programming in a typed setting. Scala 3 will
+be a big step towards realizing the full potential of these ideas. Its main objectives are to
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behavior,
+- build on strong foundations to ensure the design hangs well together,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our eighth scheduled release according to our [6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+The [previous technology preview](https://github.com/lampepfl/dotty/releases/tag/0.7.0-RC1) simplified
+enums, introduced erased terms, improved IDE support and improved pattern matching for GADT.
+
+## What’s new in the 0.8.0-RC1 technology preview?
+
+### sbt 1 support [#3872](https://github.com/lampepfl/dotty/pull/3872)
+Starting with Dotty 0.8.0, we will only support versions of sbt >= 1.1.4. Migrating to sbt 1
+lets us use the new improved incremental compiler for Scala called [Zinc](https://github.com/sbt/zinc),
+and enables integration with tools such as [Bloop](https://scalacenter.github.io/bloop/).
```

Comment:
How about:

> Dotty 0.8.0-RC1 is now compatible with Zinc 1.x. Migrating to the latest version of Zinc let's us benefit from improved incremental compilation. This change means that sbt 1.1.4 or later must be used, and that integrations with tools such as Bloop are now possible.

Reply:
I prefer to put the emphasis on sbt rather than Zinc. Users don't have to know that they are using Zinc under the hood